kafka-connect-s3/src/main/java/io/confluent/connect/s3/S3SinkConnectorConfig.java

```diff
@@ -63,6 +63,11 @@
   public static final String REGION_CONFIG = "s3.region";
   public static final String REGION_DEFAULT = Regions.DEFAULT_REGION.getName();
 
+  public static final String S3_PROXY_HOST = "s3.proxy.host";
+  public static final String S3_PROXY_PORT = "s3.proxy.port";
+  public static final String S3_PROXY_HOST_DEFAULT = null;
+  public static final int S3_PROXY_PORT_DEFAULT = 0;
```

Comment:
Should be `-1` to match the default in AWS sdk: https://github.com/aws/aws-sdk-java/blob/master/aws-java-sdk-core/src/main/java/com/amazonaws/ClientConfiguration.java#L155